hello-world-master/source/components/scene/getrequesttask.brs

```diff
@@ -0,0 +1,46 @@
+sub init()
```

Comment:
Since we are using this task specifically for fetching the pokemon data, could you think of a better task name and not use a generic name like `getrequesttask.brs`?